### ISSUE TITLE
Lightning donation address

### ISSUE THREAD

**Issue by casey** [Xtrimmer/lndecode#6]
Love your tool! The formatting looks good and is very readable. I wanted to send you a little donation, but wanted to avoid on-chain fees. It would be awesome if you had a way to accept lightning donations.

**Comment by Xtrimmer** [comment]
Well if there was ever a good reason to solve an issue, this would be it! I'll look into what it would take to implement lightning donations on a static website like this.

**Comment by Xtrimmer** [comment]
As a short term solution, I've added lightning node information to the page until I can add a more complete solution. I have updated the web servers but content delivery network may take up to 48 hours to update all caches worldwide. I am seeing the new changes live locally, but I had to clear my browser cache to do so.
![image](https://user-images.githubusercontent.com/4079568/130363902-3656fe9a-5387-4c09-8ea1-f928759d9dbc.png)
